pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -755,6 +759,8 @@ stdenv.mkDerivation (finalAttrs: {
           --replace-fail "from sphinx.util import status_iterator" "from sphinx.util.display import status_iterator"
       '';
 
+      debugEnabled = enableDebug;
+
```

Comment:
Is this needed?

Reply:
Not likely, removing and pushing with `-f`